app/services/new_schedule_builder.rb

```diff
@@ -2,6 +2,8 @@ class NewScheduleBuilder
 
   def self.create_empty_schedule
     Schedule.new.tap do |schedule|
+      schedule.date = Date.tomorrow
```

Comment:
Oh nice, this is great. I opened an issue to suggest defaults for week and day as well, but I'm thinking its utility might get outdated soon.